### PR TITLE
Some cEUR invite fixes

### LINKED ISSUE #977: cEUR invitation bugs
### Current behavior
1. Sending cEUR to someone new to Celo gives the incorrect invite message "... I just sent you X Celo Dollars". 
2. Pending invite shows incorrect value sent (calculated exchange rate was 1 cEUR = 1 cUSD). 

### Desired behavior
Message should state Celo Euros have been sent. Amount should be what the user specified in cEUR.

### Steps to Reproduce
1. Send someone new to the network 1 cEUR
2. Observe message text
3. Click "View All" on "Pending Invitations" banner
4. Observe that the sent amount shows `€0.85` (should be `1.00€`)
5. Notice that this value is inconsistent with the Recent transaction amount

### Versions

Valora 1.17.0
Android 
Samsung Galaxy S20

### PR DESCRIPTION
**Pull request by gnardini**
### Description

Add support for cEUR invites in pending invites. The pending invites flow (showing, reminding and reclaiming) had cUSD hard-coded, now it correctly uses the invite token.

### Other changes

N/A

### Tested

Manually

### How others should test

- Invite someone using cEUR. 
- See that the invite in the pending invitations list shows up with the correct amount.
- Check that the copy when pressing 'Remind' says 'Celo Euros'
- Check that you can reclaim the invite correctly.

### Related issues

- Fixes #977

### Backwards compatibility

N/A

### CODE REVIEW

**Comment by silasbw** [comment]
Should we add tests (maybe unit or functional/e2e) for these fixes?

**Comment by gnardini** [comment]
@silasbw Yeah, good callout, sorry for being lazy about it.

I added a unit test for the UI code. It would be ideal to have tests for the escrow saga as well, but it has no tests right now, so it's not as simple to add a new one to cover cEUR. Since this is a fix that would be great to have today, do you think it's OK not to add them for now?

As for e2e tests, we don't have e2e tests for the invite flow, but it would be great to have them. Maybe we can do that in another PR as well.

I created an issue to track these testing efforts: https://github.com/valora-inc/wallet/issues/992

**Review by silasbw — APPROVED** [review]
:fire: !

A'd with the caveat that I don't understand the intended behavior (or code) except for what's defined in the unit tests.

**Review by isabellewei — APPROVED** [review]
I had trouble loading the transaction feed so wasn't able to test, but it looks good to me!

File: packages/mobile/src/escrow/EscrowedPaymentListItem.tsx [hunk 1]
```diff
@@ -33,7 +33,12 @@ function EscrowedPaymentListItem({ payment }: Props) {
     })
 
     try {
-      await Share.share({ message: t('walletFlow5:escrowedPaymentReminderSmsNoData') })
+      await Share.share({
+        message: t('walletFlow5:escrowedPaymentReminderSmsNoData', {
+          currency:
+            payment.currency === Currency.Dollar ? t('global:celoDollars') : t('global:celoEuros'),
```
[the remark this answers is not in the document]
Comment (gnardini) [reply packages/mobile/src/escrow/EscrowedPaymentListItem.tsx:39]: That enum doesn't exist anymore, it was replaced by `Currency`

**Comment by MuckT** [comment]
https://valora.testquality.com requirement [R77](https://valora.testquality.com/project/12128/testcase/297910/requirement/11660).

**Comment by ValoraQA** [comment]
Hi @MuckT We have verified this issue on Latest Android internal build V1.18.0  & Testflight build V1.18.0and updated the status in test quality tool.

Test quality tool link:


iPhone 6+(12.5.4):
- https://valora.testquality.com/project/12128/plan/21657/run/149950/result/2469277

**New issue:**
- #1016 

Verified on Devices: iPhone 6+(12.5.4)
CC: @gnardini
Thanks..!!

**Comment by ValoraQA** [comment]
Hi @MuckT We have verified this issue on Latest Android internal build V1.18.0 & Testflight build V1.18.0and updated the status in test quality tool.

Test quality tool link:

iPhone 6+(12.5.4):

https://valora.testquality.com/project/12128/plan/21657/run/149950/result/2469277
New issue:
- #1020 
Verified on Devices: iPhone 6+(12.5.4)
CC: @gnardini
Thanks..!!

**Comment by ValoraQA** [comment]
Hi @MuckT We have verified this issue on Latest  Testflight build V1.18.0 and updated the status in test quality tool.


**Test quality tool link:**
- https://valora.testquality.com/project/12128/plan/21657/run/149950/result/2469266

**Verified On Device:** iPhone SE (13.6.1)
CC:  @gnardini 
Thanks..!!

**Pull request merged**